cmd/admin-handlers.go

```diff
@@ -478,6 +483,90 @@ func (a adminAPIHandlers) GetConfigHandler(w http.ResponseWriter, r *http.Reques
 	writeSuccessResponseJSON(w, econfigData)
 }
 
+// Disable tidwall json array notation in JSON key path so
+// users can set json with a key as a number.
+// In tidwall json, notify.webhook.0 = val means { "notify" : { "webhook" : [val] }}
+// In Minio, notify.webhook.0 = val means { "notify" : { "webhook" : {"0" : val}}}
+func normalizeJSONKey(input string) (key string) {
```

Comment:
Can this function move to pkg/quick ?

Reply:
> Can this function move to pkg/quick ?

I think this is very specific to Set Config Keys keys name format, I don't see how it can fit in pkg/quick package.

Reply:
Ah i see.. we don't need gjson dependency in quick.